src/cc/arduino/plugins/wifi101/flashers/java/SSLCertDownloader.java

```diff
@@ -82,7 +82,7 @@ public void checkServerTrusted(X509Certificate[] xcs, String string) throws Cert
 		connection.setSSLSocketFactory(ssl.getSocketFactory());
 
 		Certificate[] certificates = null;
-		if (connection.getResponseCode() == 200) {
+		if (connection.getResponseCode() > 0) {
```

Comment:
Let's go with the current changes, otherwise you get this exception:

```
java.lang.IllegalStateException: connection not yet open
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getServerCertificates(AbstractDelegateHttpsURLConnection.java:256)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getServerCertificates(HttpsURLConnectionImpl.java:197)
	at cc.arduino.plugins.wifi101.flashers.java.SSLCertDownloader.retrieveFromURL(SSLCertDownloader.java:84)
	at cc.arduino.plugins.wifi101.flashers.java.JavaFlasher.createBundleFromWebsites(JavaFlasher.java:168)
	at cc.arduino.plugins.wifi101.flashers.java.JavaFlasher.uploadCertificates(JavaFlasher.java:126)
	at cc.arduino.plugins.wifi101.UpdaterImpl$4.run(UpdaterImpl.java:198)
```